payload.go

```diff
@@ -137,7 +144,15 @@ func ProcessPayload(e []github.Event, u User) ([]ActivityPayload, error) {
 			if comment, ok := rawPayload["comment"].(map[string]interface{}); ok {
 				payload.Body = comment["body"].(string)
 			}
-			activityPayloads = append(activityPayloads, payload)
+
+			// Save the paylod to the db events table
+			err := payload.Create()
+			if err != nil {
+				//Assuming fail due to the unique constraint
+			} else {
+				activityPayloads = append(activityPayloads, payload)
+			}
+
```

Comment:
Oh.. I see what you're saying. For some reason I was thinking we would still append the payload to the activityPayloads slice. But you're right. I'll just put this whole block after the switch